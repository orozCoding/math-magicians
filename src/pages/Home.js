import React from 'react';
import './Pages.css';
import { text1, text2 } from '../components/HomeText';

const Home = () => (
  <div className="homeContainer d-flex col">
    <p className="homeTitle">
      Welcome to our page!
    </p>
    <p className="text">{text1}</p>
    <p className="text">{text2}</p>
  </div>
);

export default Home;
