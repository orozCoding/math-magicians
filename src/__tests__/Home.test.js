import React from 'react';
import '@testing-library/react';
import renderer from 'react-test-renderer';
import Home from '../pages/Home';

describe('Renders Home', () => {
  it('renders', () => {
    const tree = renderer
      .create(
        <Home />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
