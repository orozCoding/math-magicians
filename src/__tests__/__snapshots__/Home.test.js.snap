// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Home renders 1`] = `
<div
  className="homeContainer d-flex col"
>
  <p
    className="homeTitle"
  >
    Welcome to our page!
  </p>
  <p
    className="text"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc vitae purus lorem. Nulla facilisi. Nunc convallis nibh sem, at mollis elit gravida vel. Nulla faucibus ac felis at facilisis. Nullam in quam sollicitudin tellus ornare fermentum. Aenean viverra lectus id vehicula volutpat. Ut vulputate quam ac sem volutpat tempor. Donec nec urna justo. Vivamus nulla orci, fermentum id feugiat quis, condimentum nec est. Integer tristique suscipit augue pellentesque dapibus. Sed dapibus aliquam urna, sed vulputate mi gravida et. Ut ac imperdiet purus, nec pretium libero.
  </p>
  <p
    className="text"
  >
    Ut id nisl a dolor scelerisque fermentum. Sed sed ipsum ex. Vestibulum eu nisi at nunc ornare laoreet. Fusce id nisl ac leo porttitor pretium eget eu enim. Proin diam lectus, malesuada lobortis ultrices vel, rutrum ut odio. Fusce nulla sem, fermentum at ultricies a, pretium non nunc. Vivamus ultrices tempor elementum. Integer at magna porta, hendrerit nulla ac, pretium est. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce eleifend sem non ex ornare ultrices. Vestibulum augue metus, congue non fringilla ac, fermentum id leo. In egestas dignissim nulla sit amet feugiat. Vivamus nulla enim, imperdiet vel nunc vitae, ultricies porttitor orci. Proin placerat egestas neque et suscipit. Sed sagittis pulvinar diam. Curabitur commodo, erat at finibus molestie, ligula nibh egestas velit, bibendum venenatis nulla tellus in leo.
  </p>
</div>
`;
